Src/LabHostConfiguration.ps1

```diff
@@ -155,7 +155,7 @@ function Start-LabHostConfiguration {
 		$hostDefaults = GetConfigurationData -Configuration Host;
         foreach ($property in $hostDefaults.PSObject.Properties) {
             if (($property.Name.EndsWith('Path')) -and (-not [System.String]::IsNullOrEmpty($property.Value))) {
-                [ref] $null = NewDirectory -Path $property.Value -ErrorAction Stop;
+                [ref] $null = NewDirectory -Path $(ResolvePathEx -Path $Property.Value) -ErrorAction Stop;
```

Comment:
@iainbrighton Yes ! I noticed it right after I created the PR. I have it fixed in my local repo, will cancel this PR and create another one with fixes for both :
Start-LabHostConfiguration
Test-LabHostConfiguration